src/pcf_flags.h

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file pcf_flags.h
+ *
+ * @author  Guy Maurel
+ * @license GPL v2+
+ */
+
+#ifndef PCF_FLAGS_STR_INCLUDED
+#define PCF_FLAGS_STR_INCLUDED
+
+#include "base_types.h"
+
+#include "enum_flags.h"
+#include "log_levels.h"
+#include "logger.h"
+
+#include <vector>      // do get the definition of size_t
```

Comment:
BTW, according to https://en.cppreference.com/w/c/types/size_t, `<cstddef>` should also suffice, and is probably lighter.